server/src/test/java/org/elasticsearch/action/search/SearchResponseTests.java

```diff
@@ -119,7 +119,7 @@ private SearchResponse createTestItem(boolean minimal, ShardSearchFailure... sha
                 InternalAggregations aggregations = aggregationsTests.createTestInstance();
                 Suggest suggest = SuggestTests.createTestItem();
                 SearchProfileResults profileResults = SearchProfileResultsTests.createTestItem();
-                return new SearchResponse(
+                return SearchResponse.newWithoutMetrics(
```

Comment:
I wonder if additionally to the integ test you added, we should also have an additional unit test in this class that verifies that the constructor of SearchResponse stores the took metric. What do you think? It would be more contained than an integ test, but an integ test is still great to have to verify that all the pieces integrate well with each other.